packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -524,7 +525,9 @@ export class DatabaseCluster extends DatabaseClusterNew {
     }
 
     setLogRetention(this, props);
-    createInstances(this, props, this.subnetGroup);
+    const { instanceIdentifiers, instanceEndpoints } = createInstances(this, props, this.subnetGroup);
+    this.instanceIdentifiers = instanceIdentifiers;
+    this.instanceEndpoints = instanceEndpoints;
```

Comment:
Super minor, but I feel like this is a little bit repetitive. How about

```suggestion
    const createdInstances = createInstances(this, props, this.subnetGroup);
    this.instanceIdentifiers = createdInstances.instanceIdentifiers;
    this.instanceEndpoints = createdInstances.instanceEndpoints;
```

?